Engine/Lut.cpp

```diff
@@ -167,11 +167,12 @@ LutManager::~LutManager()
     }
 }
 
-static bool
-clip(RectI* what,
-     const RectI & to)
+static RectI
+clip(const RectI& what,
```

Comment:
This clips twice. Maybe call it clip2 to make it clear that this one is called?

Reply:
Done. Thanks for the review.